src/entities/book.rb

```diff
@@ -0,0 +1,19 @@
+require_relative "entity"
+
+class Book < Entity
```

Comment:
You can redefine default `to_s` method for all entities, for example
```
  def to_s
    "#{@title}, #{@author}"
  end
```

Reply:
done